language/stdlib/modules/references_template.md

```diff
@@ -1,2 +1,2 @@
-
-[]: # (File containing markdown style reference definitions to be included in each generated doc)
+[ROLE]: https://github.com/libra/libra/blob/master/language/move-prover/doc/user/access-control.md#roles
```

Comment:
So we are planning to link only to the table as a whole? Probably OK given that this tables aren't very large.

Reply:
I think it's OK too. The "roles" table is small enough to fit in a page/screen. The "permissions" table can be a bit over one page/screen, but it may be still good to link the top of the table where the column labels (e.g., H, I, J) are visible. @sblackshear , @tzakian , do you have an opinion on this?

Reply:
Let's not block on this waiting for Sam and Tim, I suggest submit and we can still change it later if desired.